tests/test_analytical_models.py

```
@@ -0,0 +1,1014 @@
+        return model, params
+
+    def test_shape(self):
+        model, params = self._make_model_and_params()
+        times, mags = model.predict(params)
+        assert times.shape == (100,)
+        for filt in FILTERS:
+            assert mags[filt].shape == (100,), f"Wrong shape for {filt}"
+
+    def test_finite(self):
+        model, params = self._make_model_and_params()
+        times, mags = model.predict(params)
```

Ruff (0.15.2)

[warning] Unpacked variable `times` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)